leesy/index/home.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+    <meta charset="UTF-8">
+    <title> leesy ruiz </title>
+    <link href="/Users/lisyruiz/Documents/Ada/6_week/Static-Site/leesy/style/home.css" rel="stylesheet" type="text/css">
+    <link href="/Users/lisyruiz/Documents/Ada/6_week/Static-Site/leesy/style/normalize.css" rel="stylesheet" type="text/css">
+
+</head>
+
+<body>
+    <header>
+
+        <nav class="nav_container">
+
+
+            <div class="page_link">
+                <a href="index/portfolio.html"> portfolio </a>
+            </div>
+
+            <div class="page_link">
+                <a href="index/blog.html"> blog </a>
+            </div>
+
+            <div class="page_link">
+                <a href="index/aboutme.html"> about me </a>
+            </div>
+
+            <div id="swipe">
+                <div>💋&nbsp;&nbsp;&nbsp;&nbsp;swipe right </div>
+            </div>
+
+
+        </nav>
+
+    </header>
+
+
+
+    <div id="slide" class="slide1">
+
+        <p id="logo">
```

Comment:
I'm not sure what this is, so I can't say for sure, but I don't think its a paragraph.